src/Modal.tsx

```diff
@@ -195,7 +195,7 @@ export class Modal extends Component<ModalProps, ModalState> {
         </TouchableWithoutFeedback>
         {visible && (
           <Animated.View
-            style={[styles.content, contentAnimatedStyle]}
+            style={[styles.content, contentAnimatedStyle, contentStyle]}
```

Comment:
This was destructured above so I thought it was supposed to be used here?

Reply:
Yeah, I think we skipped it in an older PR because we didn't want to expose it but then we reverted it. So yes, it was supposed to be here 👍